### ISSUE TITLE
Support `-p`/`--package` argument

### ISSUE THREAD

**Issue by jmesmon** [cmyr/cargo-instruments#56]
Right now, I have to `cd` into a particular package within my workspace to use `cargo-instruments`. Other builtin cargo commands (like `cargo run` and `cargo build`) support a `-p`/`--package` argument so that one doesn't need to `cd` around. It would be good for `cargo-instruments` to support the same option.

**Comment by cmyr** [comment]
Yes, this would be nice. I actually hacked on this for a while, and it ended up not being trivial, so I didn't follow up; basically to do this correctly we would want to pretty closely match the behaviour of the built-in `cargo` tools, which means doing this like respecting the `default-run` and `default-members` manifest fields, and doing *this* requires a fairly large-scale rearchitecture. 😒
